pkg/ui/src/views/cluster/components/range/index.tsx

```diff
@@ -126,7 +126,28 @@ class RangeSelect extends React.Component<RangeSelectProps, RangeSelectState> {
 
   toggleCustomPicker = (custom: boolean) => () => this.setState({ custom }, this.clearPanelValues);
 
-  toggleDropDown = () => this.setState({ opened: !this.state.opened }, this.toggleCustomPicker(this.state.opened ));
+  toggleDropDown = () => {
+    this.setState(
+      (prevState) => {
+        return {
+          opened: !prevState.opened,
+          /*
+          Always close the custom date picker pane when toggling the dropdown.
+
+          The user must always manually choose to open it because right now we have
+          no button to "go back" to the list of presets from the custom timepicker.
+           */
+          custom: false,
+        };
+      },
+      () => {
```

Comment:
there is no `onOpened` and `onClosed` props yet here.

Reply:
oh no! 🤒 This explains why I got a merge conflict when making this change. thankyou I will revise.